### ISSUE TITLE
Example Request - A script that Adds text to chunk title or right-click menu

### ISSUE THREAD

**Issue by ahalota** [agisoft-llc/metashape-scripts#19]
I really like the templates for creating GUI scripts you have made, is it possible to create  few more to show how to put items elsewhere in the GUI than the main menu?

I'd really like to add something to the right-click menu for a chunk, or to be able to add custom info to the chunk label (I need the total chunk error displayed on the main label, not in the reference panel).

A few basic examples or the API for adding things to different parts of the GUI would be great. Thanks!

**Comment by PolarNick239** [comment]
Currently this is not possible. There is only ```addMenuItem```.

To show total chunk error in the chunk label - you can as usual add custom menu button ```Update chunk errors``` which will append total error to ```chunk.label```.

**Comment by ahalota** [comment]
Ah, too bad.

I was trying to figure out the QtWidgets and navigate through them to find the piece where additional items are displayed (points, [R], etc.) , but it's hard to follow that structure just by playing around in the python console. It does seem like it would be possible, if I knew more about the layout of the items in the GUI.

So far I have been doing that, yes, updating the labels. It's been great being able to add GUI tools for quickly running my code, thank you for that feature.
